docs/data_prep.md

```diff
@@ -5,30 +5,56 @@
     We recommend also checking out [Nipoppy](https://nipoppy.readthedocs.io/), a protocol for standardized organization and processing of clinical-neuroimaging datasets that extends [BIDS](https://bids-specification.readthedocs.io/en/stable/). 
     Neurobagel tools are designed to be compatible with data organized according to the Nipoppy specification, although you do not need to use Nipoppy in order to use Neurobagel.
 
-To use the Neurobagel annotation tool, 
+To use the Neurobagel annotation tool,
 please prepare the tabular data for your dataset as a single, tab-separated file (`.tsv`).
 
 !!! note
     In the Neurobagel context, _tabular_ or _phenotypic_ data for a dataset refers to any demographic,
-    clinical/behavioural, cognitive, or other non-imaging-derived data of participants 
+    clinical/behavioural, cognitive, or other non-imaging-derived data of participants
     which are typically stored in a tabular file format.
 
 ## General requirements for the phenotypic TSV
 
 ### All datasets
 
-A valid dataset for Neurobagel **must** include a TSV file that describes participant attributes. 
-The TSV must contain a minimum of two columns: at least one column must contain subject IDs, 
-and at least one column must describe demographic or other phenotypic information 
-(for variables currently modeled by Neurobagel, see the [data dictionary section](dictionaries.md)).
+A valid dataset for Neurobagel **MUST** include a TSV file that describes participant attributes.
+
+The TSV MUST contain:
+
+- A minimum of two columns
+- At least one column containing subject IDs
+
+    ??? note "Only one subject ID column can be annotated"
+        Neurobagel currently does not support annotating multiple subject ID columns
+        so you must choose one as the primary ID during annotation
+
+- At least one column that describes demographic or other phenotypic information
+- Unique values in the subject ID column or unique combinations of IDs if both subject and session ID columns are present
+
+The TSV MAY contain:
+
+- A column with session IDs, e.g. if the dataset is longitudinal
+  
+    ??? note "Only one session ID column can be annotated"
+        Neurobagel currently does not support annotating multiple session ID columns
+        so you must choose one as the primary ID during annotation
+
+The TSV MUST **NOT** contain:
+
+- Missing values in the subject ID and session ID (if available) columns
```

Comment:
Should we add this clarification here for datasets w/ multiple ID columns where some may have missing values?

```suggestion
- Missing values in the columns you plan to annotate as containing the primary subject IDs and session IDs (if available)
```

Not sure if that helps - feel free to ignore if you think so.